sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveSparkSubmitSuite.scala

```diff
@@ -838,3 +838,19 @@ object SPARK_18989_DESC_TABLE {
     }
   }
 }
+
+object SPARK_27814 {
+  def main(args: Array[String]): Unit = {
+    val spark = SparkSession.builder().enableHiveSupport().getOrCreate()
+    try {
+      spark.sql("DROP TABLE IF EXISTS t1")
```

Comment:
can you move it to `org.apache.spark.sql.hive.execution.SQLQuerySuite`?

Reply:
thanks. I will try.